src/NuGet.Clients/VsConsole/Console/ScriptExecutor.cs

```diff
@@ -175,7 +175,18 @@ public bool TryMarkVisited(PackageIdentity packageIdentity, PackageInitPS1State
             // to switch to powershell pipeline execution thread. In order not to block the UI thread,
             // go off the UI thread. This is important, since, switches to UI thread,
```

Comment:
This comment is now contradictory to your code change and out of date. Can you update it?